### ISSUE TITLE
Misleading parse error

### ISSUE THREAD

**Issue by dirk-zimoch** [w3c/css-validator#520]
The validator claims a  "Parse Error" at the word `after` in the following (perfectly working) CSS rule:
```
a[href]:where(
  [href^="https://"],
  [href^="http://"],
):after {
    content: url('open-external-link-icon.svg');
    display: inline-block;
    width:1.2ex;
    margin-left:.3ex;
}
```

**Comment by dirk-zimoch** [comment]
The problem is the trailing `,` after the last `where` condition. The error message is misleading in relation to the location of the error.
